vocab/vocabulary.yml

```diff
@@ -0,0 +1,91 @@
+vocab:
+  - id: sl
```

Comment:
Would it be user friendly to chose a different prefix, say, `slist`? The problem is that it is difficult to differentiate between `s1` and `sl` in many fonts, and this may be the source of stupid bugs...

Reply:
(I made the mistake reviewing this file in github: I was wondering why on earth do we use the prefix `s1`, ie, "sh one"...)

Reply:
Now for the important question... if not `sl`, then what? :P 

Would `sv` be better? For "Status Vocabulary"? and then perhaps shorten the URL to: `https://www.w3.org/ns/credentials/status#` ?

Reply:
I think `sv` is really better.

As for the URL, for the reasons I started this discussions with, my personal preference is to put it into /2018/, alongside the VCDM vocabulary.

Reply:
(but shorten it to `.../status` is indeed a good idea imho.)

Reply:
Just to be precise, because I made a mistake in the original comment, what about `https://www.w3.org/2018/credentials/status#`

(Noting that we will have to update the `.htaccess` file in `/2018/credentials/`)

Reply:
Per [WG meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-01-10-vcwg#section3-8) this issue is getting closed. But I think using "sv" instead of "sl" would still be better, and that means that URL would be `/ns/credentials/status#`.

Once this is merged, the `.htaccess` in `/ns/credentials` will need some magic...

Reply:
Updated in 636451dd7676b2922905dc919695054fec0af6e6